README.md

```diff
@@ -48,20 +51,24 @@ This is sometimes easier said than done.
 Sharing these research outputs means understanding data management, library sciences, software development, and continuous integration techniques: skills that are not widely taught or expected of academic researchers and data scientists. As these activities are not commonly taught, we recognise that the burden of requirement and new skill acquisition can be intimidating to individuals who are new to this world.
 _The Turing Way_ is a handbook to support students, their supervisors, funders and journal editors in ensuring that reproducible data science is "too easy not to do" even for people who have never worked in this way before.
 It will include training material on version control, analysis testing, and open and transparent communication with future users, and build on Turing Institute case studies and workshops.
-This project is openly developed and any and all questions, comments and recommendations are welcome at our github repository: <https://github.com/alan-turing-institute/the-turing-way>.
+This project is openly developed and any and all questions, comments and recommendations are welcome at our GitHub repository: <https://github.com/alan-turing-institute/the-turing-way>.
+
+### The Team
 
-### The team
+_The Turing Way_ a community-led project.
+Everyone who contributes to this book, no matter how small or big their contributions are, is recognised in this project as a contributor and a community member. 
+Long-term contributors of the project are considered part of the core contributors groups who take on various leadership roles in the project.
 
-This is (part of) the project team planning work at the Turing Institute.
-For more on how to contact us, see the [ways of working document](ways_of_working.md).
+The project is coordinated by the lead investigators **Kirstie Whitaker** (founder) and **Malvika Sharan** (community developer) and hosted at [The Alan Turing Institute](https://www.turing.ac.uk/).
 
-![Team photo](book/website/figures/theturingway-team.jpg)
+You can read [_The Turing Way_ acknowledgement process](https://the-turing-way.netlify.app/community-handbook/acknowledgement/acknowledgement-members.html#community-members-contributors-and-co-authors) and [Record of Contributions](https://the-turing-way.netlify.app/afterword/contributors-record.html) to learn about how we acknowledge your work and how our contributors are highlighted in the project.
```

Comment:
Is there a reason you've chosen to not use the [jupyter book cross-referencing as recommended in the style guide](https://the-turing-way.netlify.app/community-handbook/style/style-crossref.html)?

```suggestion
You can read [_The Turing Way_ acknowledgement process](https://the-turing-way.netlify.app/community-handbook/acknowledgement/acknowledgement-members.html#community-members-contributors-and-co-authors) and {ref}`contributors-record-highlights` to learn about how we acknowledge your work and how our contributors are highlighted in the project.
```

I notice that the first link doesn't have a label, but that could (should?) be fixed.

Reply:
The cross referencing will not work on the README file as it is not part of the book.

Reply:
Of course, silly me!